android/src/main/java/com/opentokreactnative/utils/EventUtils.java

```diff
@@ -21,7 +21,7 @@ public static WritableMap prepareJSConnectionMap(Connection connection) {
         return connectionInfo;
     }
 
-    public static WritableMap prepareJSStreamMap(Stream stream) {
+    public static WritableMap prepareJSStreamMap(Stream stream, Session session) {
```

Comment:
We're passing the complete session object just to use the sessionId. Should we just pass the sessionId to prepareJSStreamMap?

Reply:
Just passing in the sessionId should work for this update. If there's any plans in the future to access more methods from the session object, then might be better to keep it as is

Reply:
I would stick with the Session object as it's more flexible for future changes (as Allen said if we want to add more properties). @ggoldens do you see any downside on passing the entire object?

Reply:
Was just a suggestion. I'm fine with passing the entire object if you believe that in the future we will use it.
Thanks!